music21/noteworthy/translate.py

```diff
@@ -840,6 +972,61 @@ def createLyrics(self, attributes):
                         ll = ' - '
                     lyrics.append(ll)
         return lyrics
+    def createStaff(self, attributes):
+        self.createPart()
+        self.currentKey = key.KeySignature(0)
+        self.activeAccidentals = {}
+        self.lyrics = []
+        self.lyricPosition = 0
+        self.currentPart.partName = attributes.get('Name')
+        self.currentPart.partAbbreviation = attributes.get('Name')
+
+    def createStaffInstrument(self, attributes):
+        patch = int(attributes.get('Patch', 0))
+        try:
+            staffInstru = instrument.instrumentFromMidiProgram(patch)
+            readTranspo = int(attributes.get('Trans', 1))
+            self.currentTransposition = 12 * readTranspo // 12
+            staffInstru.transposition = interval.Interval(readTranspo)
+            self.currentPart.append(staffInstru)
+        except InstrumentException:
+            for p in range(34, 81):
+                i = instrument.unpitchedInstrumentFromPercMapPitch(p)
+                i.midiProgram = 0
+                i.midiChannel = 9
+                self.currentPart.append(i)
+            self.currentPart.partName = 'Drumset'
+            self.currentPart.partAbbreviation = 'D.'
+
+    def createText(self, attributes):
+        text = attributes['Text']
+        # Rehearsal is associated to the score while TextExpression is staff-associated.
+        expr = expressions.TextExpression(text)
+        expr.positionPlacement = 'above' if int(attributes.get('Pos')) < 0 else 'below'
+        expr.style.relativey = attributes.get('Pos')
+        self.currentMeasure.append(expr)
+
+    def createSongInfo(self, attributes):
+        title = attributes.get('Title')
+        author = attributes.get('Author')
+        md = metadata.Metadata()
+        md.title = title
+        md.author = author
+        self.score.append(md)
+
+    def createEnding(self, attributes):
+        end = attributes['Endings']
+        if self.currentMeasure.leftBarline is None:
+            self.currentMeasure.leftBarline = bar.Barline()
+        self.repeatedMeasures = [self.currentMeasure]
+        endingType = end.split(',')[0] 
```

Comment:
Reason we just need the first value? `RepeatBracket` can take a comma-separated list of endings.

Reply:
As far as I understood, first value is the type and correspond to `RepearBracket.number` and (optional) second is the 'type', but I didn't find a correct doc about it, so I let it unused